modyn/supervisor/internal/triggers/amounttrigger.py

```diff
@@ -1,4 +1,4 @@
-from modyn.supervisor.internal.trigger import Trigger
+from modyn.supervisor.internal.triggers.trigger import Trigger
```

Comment:
if we are in the process of refactoring this, maybe export all triggers in the triggers init py and write

from modyn.supervisor.internal.triggers import Trigger

Reply:
Done